### ISSUE TITLE
Cannot find reflect and zone

### ISSUE THREAD

**Issue by yverenoir** [fabiandev/angular2-quiz-app#15]
Hi guys,

thanks for providing this fancy quiz app. I am having issues running it due to this problem:

zone.js:1 Uncaught SyntaxError: Unexpected token <
Reflect.js:1 Uncaught SyntaxError: Unexpected token <
(index):52 Error: reflect-metadata shim is required when using class decorators

It seems like the app cannot find the files and the index file is served instead. I am experiencing it both in dev (fresh from git) and prod (after I modified the app), but it works in prod if I run it fresh from git.

I would appreciate any help, thanks!

Best regards,
Yve

**Comment by fabiandev** [comment]
There was a bug in the index.html, that tried to load zone.js and reflect-metadata from node_modules via script tags. I've removed these packages a while ago as dependencies, as they are installed via jspm anyway, which lead to this issue.

I've just pushed v1.8.1 where this should be fixed. It now loads the packages via SystemJS and from jspm_packages before bootstrapping the app.

If you checkout the latest version of this repo, both, the dev and the production server should work again.

**Comment by fabiandev** [comment]
There was a bug in the index.html, that tried to load zone.js and reflect-metadata from node_modules via script tags. I've removed these packages a while ago as dependencies, as they are installed via jspm anyway, which lead to this issue.

I've just pushed v1.8.1 where this should be fixed. It now loads the packages via SystemJS and from jspm_packages before bootstrapping the app.

If you checkout the latest version of this repo, both, the dev and the production server should work again.
